### PR TITLE
[PICA-317] Fix processor crashing when attempting to store a null value on asset prices

### PR DESCRIPTION
**Pull request by andresmechali**

### CODE REVIEW

**Comment by KaiserKarel** [comment]
Task linked: [PICA-317 Subsquid: fix processor crashing every few days](https://app.clickup.com/t/20465559/PICA-317)

File: subsquid/src/dbHelper.ts [hunk 1]
```diff
@@ -566,21 +566,25 @@ export async function getCurrentAssetPrices(ctx: Context | EntityManager): Promi
...
+              id: randomUUID(),
+              assetId,
+              price: res[assetInfo.coingeckoId].usd,
+              timestamp: date,
+              currency: Currency.USD
+            })
+          );
+        }
+      } catch (err) {
+        console.log(err);
```
Comment (fl-y) [inline subsquid/src/dbHelper.ts:587]: Can we log this so that it's more visible maybe?

**Review by fl-y — APPROVED** [review]
LGTM

**Review by bengalmozzi — APPROVED** [review]
👍

**Review by kkast — APPROVED** [review]

**Comment by andresmechali** [comment]
@mergify refresh

**Review by andyjsbell — APPROVED** [review]
LGTM

**Review by vimukthi-git — APPROVED** [review]

**Pull request merged**